homeassistant/components/light/flux_led.py

```diff
@@ -191,8 +191,16 @@ def hs_color(self):
     @property
     def supported_features(self):
         """Flag supported features."""
+        if self._mode is MODE_RGBW:
+            return SUPPORT_FLUX_LED | SUPPORT_WHITE_VALUE
+
         return SUPPORT_FLUX_LED
 
+    @property
+    def white_value(self):
+        """Return the white value of this light between 0..255."""
+        return self._bulb.getRgbw()[3]
```

Comment:
Currently, the light entity will request the `white_value` even if not `SUPPORT_WHITE_VALUE`. Will this raise an error ?

Reply:
Retrieving index 3 should not directly result in an error because the flux_led will always return something on that index even if `None`.  So it would depend on how the code calling white_level handles the values.  I can add a sanity check if `None` or non integer values are not supported.

Reply:
If it's `None`, then it's fine as that is the expected return value for "no value"